qcodes/instrument_drivers/AlazarTech/ATS9870.py

```diff
@@ -96,16 +100,16 @@ def __init__(self, name, **kwargs):
                                             1.0: 10,
                                             2.0: 11,
                                             4.0: 12})
-            self.add_parameter(name='impedance' + i,
+            self.add_parameter(name=f'impedance{i}',
                                parameter_class=TraceParameter,
-                               label='Impedance channel ' + i,
+                               label=f'Impedance channel {i}',
                                unit='Ohm',
                                initial_value=50,
                                val_mapping={1000000: 1,
```

Comment:
This is not related to this PR, but we don't have impedance of 1000000 ohm for this card. It only supports a fixed 50 ohm. Since this parameter is not settable, it is fine but maybe it is nice to just delete this line.